### PR TITLE
VEN-109 | Boat length for boats stored on a trailer

### PR DESCRIPTION
**Pull request by midori0507**
- Sync state between `boat_has_trailer` checkbox with same radio option in step 3 of form
- Toggle text base on redux state (slow) but guarantee safe
- Fix minor text overflow along

### CODE REVIEW

File: src/components/forms/fragments/UnRegisteredBoatDetails.tsx [hunk 1]
```diff
@@ -14,16 +14,16 @@ interface Props {
   fieldsNotRequired?: boolean;
   boatTypes?: BoatTypes;
   showBoatStorageType?: boolean;
+  boatStorageType?: boolean;
```
Comment (igordavydsson) [inline src/components/forms/fragments/UnRegisteredBoatDetails.tsx]: a bit misleading: the name has word "type", but the value is boolean. maybe change the name to a more descriptive one?

Comment (midori0507) [reply src/components/forms/fragments/UnRegisteredBoatDetails.tsx]: i name it after ticket description, it should be `boatHasTrailer` ?

Comment (midori0507) [reply src/components/forms/fragments/UnRegisteredBoatDetails.tsx]: `boatIncludeTrailer` ?

Comment (igordavydsson) [reply src/components/forms/fragments/UnRegisteredBoatDetails.tsx]: `boatStoredOnTrailer` would be the most accurate 🙂

Comment (igordavydsson) [reply src/components/forms/fragments/UnRegisteredBoatDetails.tsx]: would be dev friendly for the future devs to rename the const in utils too

Comment (midori0507) [reply src/components/forms/fragments/UnRegisteredBoatDetails.tsx]: Done!

**Pull request merged**